integrationtests/src/test/java/org/neo4j/procedure/UserFunctionIT.java

```diff
@@ -712,14 +713,26 @@ public void shouldListAllFunctions() throws Throwable
         "| 'org.neo4j.procedure.simpleArgument'                | 'org.neo4j.procedure.simpleArgument(someValue :: INTEGER?) :: (INTEGER?)'                                                                                    | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareDouble'                  | 'org.neo4j.procedure.squareDouble(someValue :: FLOAT?) :: (FLOAT?)'                                                                                          | ''                      |" + lineSeparator() +
         "| 'org.neo4j.procedure.squareLong'                    | 'org.neo4j.procedure.squareLong(someValue :: INTEGER?) :: (INTEGER?)'                                                                                        | ''                      |" + lineSeparator() +
+        "| 'org.neo4j.procedure.sum'                           | 'org.neo4j.procedure.sum(numbers :: LIST? OF NUMBER?) :: (NUMBER?)'                                                                                          | ''                      |" + lineSeparator() +
```

Comment:
This is useful but I have mixed feelings exposing a function with the same name (even in  a namespace) as a built in aggregate function. Especially given that we've said we want to look into how aggregate functions might be called on lists. Could we rename this to make it a bit different from ordinary `sum`?